src/services/inplayer.account.service.ts

```diff
@@ -52,6 +52,34 @@ export const getUser = async (): Promise<Customer> => {
 
 export const getFreshJwtToken = async ({ auth }: { auth: AuthData }) => auth;
 
+export const updateCustomer: UpdateCustomer = async (values) => {
```

Comment:
Really nice that you can just ignore the cleeng params and the method signature works without any other chanes.